driver_test.go

```diff
@@ -2035,7 +2046,7 @@ func startDestroyInspectImage(t *testing.T, image string, taskName string) {
 		AllocID:   uuid.Generate(),
 		Resources: createBasicResources(),
 	}
-	imageID, err := getPodmanDriver(t, d).createImage(image, &TaskAuthConfig{}, false, false, 5*time.Minute, task)
+	imageID, err := getPodmanDriver(t, d).createImage(image, &TaskAuthConfig{}, false, false, getPodmanDriver(t, d).defaultPodman, 5*time.Minute, task)
```

Comment:
Nitpicky, but maybe something like:

```suggestion
	drv := getPodmanDriver(t, d)
	imageID, err := drv.createImage(image, &TaskAuthConfig{}, false, false, drv.defaultPodman, 5*time.Minute, task)
```

Similar for a lot of these below.

Reply:
Updated